packages/blocks/src/_common/inline/presets/nodes/reference-node/reference-popup.ts

```diff
@@ -10,6 +10,7 @@ import { html, LitElement, nothing } from 'lit';
 import { customElement, property, query } from 'lit/decorators.js';
 
 import type { RootBlockComponent } from '../../../../../root-block/types.js';
+import { insideEdgelessText } from '../../../../../root-block/widgets/slash-menu/utils.js';
```

Comment:
I think use `isInsideBlockByFlavour` from `../../../../utils/index.js` is better than use `inseideEdgelessText` from `slash-menu/utils.js`